cfgov/jinja2/v1/_includes/molecules/card.html

```diff
@@ -6,24 +6,34 @@
 
    Description:
 
-   Creates markup for a Card molecule, which displays an icon, some text,
-   and a link.
+   Creates markup for a Card molecule, which displays some text and a link,
+   with an optional heading and icon.
 
    value.text:                  Card body text.
 
    value.link_text:             Card link text.
 
    value.link_url:              Card link URL.
 
+   value.heading:               Card heading (optional).
```

Comment:
Other places we put `(optional)` first, but we're not consistent. See https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/jinja2/v1/_includes/molecules/text-introduction.html for example.

Reply:
Thanks, I like that better. Will change.